app/controllers/proprietor/users_controller.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+module Proprietor
+  class UsersController < ProprietorController
+    before_action :ensure_admin!
+
+    before_action :find_user, only: %i[show edit update destroy]
+    load_and_authorize_resource
+
+    # GET /users
+    # GET /users.json
+    def index
+      authorize! :manage, User
+      add_breadcrumb t(:'hyrax.controls.home'), root_path
+      add_breadcrumb t(:'hyrax.admin.sidebar.users'), proprietor_users_path
+    end
+
+    # GET /users/1
+    # GET /users/1.json
+    def show
+      add_breadcrumb t(:'hyrax.controls.home'), root_path
+      add_breadcrumb t(:'hyrax.admin.sidebar.users'), proprietor_users_path
+      add_breadcrumb @user.display_name, edit_proprietor_user_path(@user)
+    end
+
+    # GET /users/new
+    def new
+      add_breadcrumb t(:'hyrax.controls.home'), root_path
+      add_breadcrumb t(:'hyrax.admin.sidebar.users'), proprietor_users_path
+      add_breadcrumb 'New user', new_proprietor_user_path
+    end
+
+    # GET /users/1/edit
+    def edit
+      add_breadcrumb t(:'hyrax.controls.home'), root_path
+      add_breadcrumb t(:'hyrax.admin.sidebar.users'), proprietor_users_path
+      add_breadcrumb @user.display_name, edit_proprietor_user_path(@user)
+    end
+
+    # POST /users
+    # POST /users.json
+    def create
+      @user = User.new(user_params)
+      respond_to do |format|
+        if @user.valid? && @user.save
+          format.html { redirect_to proprietor_users_path, notice: 'User was successfully created.' }
+          format.json { render :show, status: :created, location: @user.cname }
+        else
+          format.html { render :new }
+          format.json { render json: @user.errors, status: :unprocessable_entity }
+        end
+      end
+    end
+
+    # PATCH/PUT /users/1
+    # PATCH/PUT /users/1.json
+    def update
+      respond_to do |format|
+        if @user.update(user_params)
+          format.html { redirect_to proprietor_users_path, notice: 'User was successfully updated.' }
+          format.json { render :show, status: :ok, location: [:proprietor, @user] }
+        else
+          format.html { render :edit }
+          format.json { render json: @user.errors, status: :unprocessable_entity }
+        end
+      end
+    end
+
+    # DELETE /users/1
+    # DELETE /users/1.json
+    def destroy
+      @user.destroy
+
+      respond_to do |format|
+        format.html { redirect_to proprietor_users_url, notice: 'User was successfully destroyed.' }
+        format.json { head :no_content }
+      end
+    end
+
+    # method uses user's id, not their user_key
+    def become
```

Comment:
Can you explain what the use case for `become` is here?  It seems like persona's become but not exactly.

Reply:
it is the same become as the one in persona - just not as full featured.  There is another PR / feature set that brings persona to the tenant admin section (giving it parity with the Avalon and direct Hyrax version).  We haven't gotten back to consolidating down the proprietor version yet, but that is work I'm hoping to do at some point i the near future.

Reply:
How do you feel about dropping `become` from this PR?  Besides not being as built out as in persona, I think it isn't core to the feature being added by this PR and I've gotten the vibe from superiors that they are apprehensive about it without more thought and/or legal review.  (I'll check with them on their reservations when I talk with them about the changes in this PR.) Maybe become could be separated out as another PR so it won't hold up this one for now?